src/software-delivery-machine/commands/editors/demo/removeTypeScriptHeader.ts

```diff
@@ -45,7 +45,7 @@ const HeaderRegex = /^\/\*[\s\S]*?\*\/\s*/;
  */
 const whackSomeHeader: SimpleProjectEditor = (p, ctx) => {
     let count = 0;
-    return doWithFiles(p, "src/**/*.ts", async f => {
+    return doWithFiles(p, "src/**/*", async f => {
```

Comment:
Please change this. It should only work on C family languages, because not all languages support the same comment syntax. Can we have a glob for Java, Scala, C, C++, Kotlin, JS, TS?

Reply:
The simple glob change doesn't work so I made the code check it.